packages/core/src/hooks/call.ts

```diff
@@ -102,8 +102,8 @@ export function useStarknetCall<T extends unknown[]>({
 
   return {
     data,
-    loading: isLoading,
-    refresh: () => undefined,
+    loading: isLoading || isRefetching,
```

Comment:
Not sure if we want the `loading` value to be true when refreshing
We can also add another "refreshing" boolean

Reply:
Actually I think that's better to add a refreshing boolean - sometimes we don't want to display data until the first load is done but we want to keep displaying it while we refresh. 
What do you think?

Reply:
Yeah I agree with the second option. That can be done in a new refactor where we change the API to be more like wagmi.

Reply:
should 'isrefetching' be 'isrefreshing' and also 'refetch' be 'refresh' or was this done on purpose?

Reply:
the old api used `refresh` while react-query calls it `refetch`.